From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@snobu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

snobu commented:
/cc @ccompy I'll e-mail you the details.

PRmerger commented:
@snobu : Thanks for your contribution to the Azure documentation! The author, @MandiOhlinger, has been notified to review your proposed change.

MandiOhlinger commented:
@snobu : Thank you for the addition. 
@ggailey777 - Let's merge as-is.

ChrisCompy commented:
@snobu: I missed your submission.  My github work account is chriscompy   my ccompy predated working at MS and I kept it separate sadly when I started at MS.  Thanks for the change. 
@MandiOhlinger :  The change is good.  Thanks for following up on it.

Commit:
update statement about ForceTakeover according to Eliza Kuzmenko

bandersmsft committed:
cost mgt - refresh connect-aws-account

grhuynh committed:
add business continuity documentation, and add link to Azure notebook to the quickstart

grhuynh committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

BryanTrach-MSFT committed:
(forumsteam) fixes MicrosoftDocs/azure-docs#6857

Updated MSA URL.

Ja-Dunn committed:
Merge pull request #37304 from neilpeterson/acs-oms-update

[AKS] oms > log analytics

Ja-Dunn committed:
Merge pull request #37306 from bandersmsft/master

cost mgt - refresh connect-aws-account

PRMerger16 committed:
Merge pull request #37305 from JasonWHowell/patch-1

Update stream-analytics-introduction.md

grhuynh committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into terraform-slot-walkthru

PRMerger18 committed:
Merge pull request #37307 from BryanTrach-MSFT/patch-14

(forumsteam) fixes MicrosoftDocs/azure-docs#6857

ShawnJackson committed:
edit pass: terraform-slot-walkthru

PRMerger15 committed:
Merge pull request #37117 from v-nagta/eliumupdate

USER STORY 1227974 SaaS App Tutorial: Elium update

Commit:
Task 902: Move up links for new domain in procedure

tamram committed:
pull changes from other PR

tamram committed:
pull changes from other PR

Commit:
Task 902: Fix formatting

Commit:
Task 1059: FIx merge issues

Commit:
Task 902: Revert to old procedure format

tamram committed:
updates for SEO for blobs

PRMerger16 committed:
Merge pull request #36332 from dksimpson/dean-1059

Task 1059: update status message for auto-validation

divyaswarnkar committed:
sb5

Commit:
Task 902: Spell out first use of CDN

Commit:
Merge pull request #37316 from dksimpson/dean-cdn

Task 902: Move up links for new domain in procedure

gsacavdm committed:
Reverting gov-cogs to correct vision endpoint

PRMerger17 committed:
Merge pull request #37318 from gsacavdm/patch-1

Reverting gov-cogs to correct vision endpoint

tamram committed:
fixing links

martincoetzer committed:
Update admin-roles-best-practices.md

PRMerger14 committed:
Merge pull request #37319 from martincoetzer/patch-4

Update admin-roles-best-practices.md

Commit:
Merge pull request #37255 from tjvishnu/patch-3

Updated File Sync Restore errors

tamram committed:
another commit

gsacavdm committed:
Add syntax highlighting

kevinvngo committed:
Removed Gen1 and Gen2

Commit:
mergeCOSDBtroubleshooting

Commit:
remote useless link

Commit:
update

Commit:
update

genlin committed:
update a term

PRMerger10 committed:
Merge pull request #37142 from nabhishek/patch-21

added Databricks Linked Service

genlin committed:
add a steps about how to reinstall Vmsnapshot ex

genlin committed:
correct typo

PRMerger14 committed:
Merge pull request #37331 from genlin/master990

backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

rayne-wiselman committed:
updating few articles

PRMerger16 committed:
Merge pull request #37332 from rayne-wiselman/rayne-april8

updating few articles

rayne-wiselman committed:
fixing few issues

rayne-wiselman committed:
Update vmware-azure-tutorial.md

MikeDodaro committed:
Expand SDK samples for visibility

MikeDodaro committed:
Correct TOC

PRMerger14 committed:
Merge pull request #37334 from rayne-wiselman/rayne=april8-2

fixing few issues

MikeDodaro committed:
move v5 version down in toc

rimman committed:
made minor changes in the Intro page

GitHubber17 committed:
corrected date typo and miss-spelling

GitHubber17 committed:
Merge pull request #36575 from v-nagta/gaggleamp

USER STORY 1221647 SaaS App Tutorial: GaggleAMP update

GitHubber17 committed:
Merge pull request #37321 from gsacavdm/patch-1

Add syntax highlighting

GitHubber17 committed:
Merge pull request #37335 from MikeDodaro/expandRefs

Expand SDK samples for visibility

Juliako committed:
Update api-management-howto-aad.md

PRMerger13 committed:
Merge pull request #37339 from Juliako/patch-4

Update api-management-howto-aad.md

Juliako committed:
Update api-management-howto-aad-b2c.md

divyaswarnkar committed:
sb6

Juliako committed:
Update api-management-howto-create-groups.md

PRMerger17 committed:
Merge pull request #37340 from Juliako/patch-4

Update api-management-howto-aad-b2c.md

PRMerger17 committed:
Merge pull request #37341 from Juliako/patch-5

Update api-management-howto-create-groups.md

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into master304

PRMerger15 committed:
Merge pull request #37245 from divyaswarnkar/servicebus

sb1

genlin committed:
correct words

Commit:
add redirect para

Commit:
add redirect para

Commit:
update redirect para

AnithaAdusumilli committed:
Update for managed disks

msvijayn committed:
Update for issue 6698

rayne-wiselman committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into rayne-march15-2

rayne-wiselman committed:
Merge branch 'rayne-march15-2' of https://github.com/rayne-wiselman/azure-docs-pr into rayne-march15-2

rayne-wiselman committed:
updating date after technical review

dominicbetts committed:
Add section on latency

Commit:
Merge pull request #37348 from dominicbetts/latency-text

Add section on latency

Rajani-Janaki-Ram committed:
fixes

Rajani-Janaki-Ram committed:
fixes

Rajani-Janaki-Ram committed:
Merge branch 'master' into master

Sogup committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

PRMerger12 committed:
Merge pull request #37351 from Sogup/patch-48

Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

dominicbetts committed:
Update diagram

PRMerger17 committed:
Merge pull request #37350 from Rajani-Janaki-Ram/master

Master

PRMerger13 committed:
Merge pull request #37344 from AnithaAdusumilli/patch-9

Update for managed disks

eross-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into lizross-what-is-azuread-update

mimig1 committed:
Cosmos DB: Experimental TOC updates per Rimma

grlima committed:
Add section on Java correlation

grlima committed:
Update application-insights-correlation.md

carolinacmoravia committed:
C15368: Line breaks needed to avoid target broken format

Hello, @mumian,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

areddish committed:
Update install instructions

Commit:
Merge pull request #37325 from kevinvngo/patch-23

Removed Gen1 and Gen2

itechedit committed:
Merge pull request #37329 from jingyanjingyan/mergetroubleshoot

mergeCOSDBtroubleshooting

itechedit committed:
Merge pull request #35468 from rayne-wiselman/rayne-march15-2

adding new troubleshooting article

itechedit committed:
Merge pull request #37330 from genlin/master304

troubleshoot-vm-by-use-nested-virtualization.md

Brenduns committed:
Servicing doc update

dominicbetts committed:
Additional changes

PRMerger14 committed:
Merge pull request #37359 from carolinacmoravia/patch-670

C15368: Line breaks needed to avoid target broken format

eross-msft committed:
added border to image

eross-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into lizross-what-is-azuread-update

rjagiewich committed:
Merge pull request #37346 from msvijayn/vijayn-docs

Update for issue 6698

eross-msft committed:
fixed typo

eross-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into lizross-what-is-azuread-update

rjagiewich committed:
Merge pull request #37365 from Brenduns/april-6

Servicing doc update

grlima committed:
Update application-insights-correlation.md

itechedit committed:
Merge pull request #37358 from mimig1/04092018TOC

Cosmos DB: Experimental TOC updates per Rimma

grhuynh committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

grhuynh committed:
fix spelling error in business continuity metadata title and capitalization in TOC entry for business continuity

grlima committed:
Update application-insights-correlation.md

itechedit committed:
Merge pull request #37363 from eross-msft/lizross-what-is-azuread-update

Updated What is Azure AD to fix incorrect intro text

Commit:
Update index.md

gatneil committed:
added missing property

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Ja-Dunn committed:
Merge pull request #37313 from ShawnJackson/terraform-slot-walkthru

edit pass: terraform-slot-walkthru

Commit:
Merge pull request #37254 from JiayueHu/master

Add IoT Central into index.md

JasonWHowell committed:
Update time-series-insights-send-events.md

itechedit committed:
Merge pull request #37308 from grhuynh/master

Add business continuity documentation, and add link to Azure notebook tutorial in quickstart

JasonWHowell committed:
Update time-series-insights-diagnose-and-solve-problems.md

dominicbetts committed:
Update diagram

PRMerger16 committed:
Merge pull request #37367 from gatneil/master

added missing property

JasonWHowell committed:
Update time-series-insights-diagnose-and-solve-problems.md

JasonWHowell committed:
Update time-series-insights-diagnose-and-solve-problems.md

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-0321

barbkess committed:
fixes from casey

Commit:
Merge pull request #37371 from JasonWHowell/patch-2

Updates to make doc more useful - time-series-insights-diagnose-and-solve-problems.md

mrbullwinkle committed:
Updates to pricing doc

derek1ee committed:
Update scheduler-concepts-terms.md

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (12)

v-alje committed:
Merge pull request #37369 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger10 committed:
Merge pull request #37375 from derek1ee/patch-6

Update scheduler-concepts-terms.md

PRMerger10 committed:
Merge pull request #37206 from cherylmc/patch-4

Update vpn-gateway-about-vpn-gateway-settings.md

PRMerger12 committed:
Merge pull request #37368 from rimman/master

minor changes in the Intro page

esung22 committed:
minor updates

JasonWHowell committed:
Update time-series-insights-send-events.md

WilliamAntonRohm committed:
editing per review comments

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (13)

mrbullwinkle committed:
small grammatical change

v-shils committed:
Merge pull request #37352 from dominicbetts/update-suite-diagram

Update diagram

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (14)

JasonWHowell committed:
Update time-series-insights-send-events.md

neilpeterson committed:
[AKS] fixes MicrosoftDocs/azure-docs#6837

mimig1 committed:
Cosmos DB: Correcting Mimi and Arramac metadata

PRMerger16 committed:
Merge pull request #37253 from craigcaseyMSFT/v-craic0406

updates for the rebranding of OMS

mimig1 committed:
Merge branch 'master' into 04092018meta

esung22 committed:
correction

JasonWHowell committed:
Update time-series-insights-send-events.md

PRMerger17 committed:
Merge pull request #37378 from mimig1/04092018meta12

Cosmos DB: Moving Mimi's files to Sneha & Kelly (12)

PRMerger17 committed:
Merge pull request #37379 from mimig1/04092018meta13

Cosmos DB: Moving Mimi's files to Sneha & Kelly (13)

PRMerger18 committed:
Merge pull request #37377 from mrbullwinkle/mrb_04_09_2018_pricing_updates

Updates to pricing doc

Commit:
updates after review

JasonWHowell committed:
Update time-series-insights-send-events.md

rjagiewich committed:
Merge pull request #37125 from craigcaseyMSFT/v-craic0404

updates for the rebranding of OMS

PRMerger20 committed:
Merge pull request #37381 from neilpeterson/aks-issue-6837

[AKS] fixes MicrosoftDocs/azure-docs#6837

cherylmc committed:
Update expressroute-faqs.md

bandersmsft committed:
cost mgt - updated connect aws account

esung22 committed:
fix links

Commit:
Merge pull request #37150 from craigcaseyMSFT/v-craic0405

updates for the rebranding of OMS

JasonWHowell committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

CarlRabeler committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Added in-page links for portal.

Commit:
Added in-page links for portal.

PRMerger10 committed:
Merge pull request #37390 from cherylmc/patch-1

Update expressroute-faqs.md

MarkMcGeeAtAquent committed:
Merge branch 'master' into table-remove-includes

parakhj committed:
Adding/Removing Notes

Commit:
Fix.

Commit:
Fix

CarlRabeler committed:
[sql db] added links

v-shils committed:
Merge pull request #37218 from cristy/cgrcosmosdb

Cgrcosmosdb

Commit:
Fix

CarlRabeler committed:
more

daveba committed:
adding cosmos db tutorial and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-linux-vm-access-cosmos-db

PRMerger15 committed:
Merge pull request #37396 from parakhj/SSO

Adding/Removing Notes for SSO and SPA articles

v-shils committed:
Merge pull request #37370 from JasonWHowell/patch-1

Adding more detail/examples - Update time-series-insights-send-events.md

ktoliver committed:
Merge pull request #37271 from WilliamAntonRohm/W-oms-07

OMS rebranding -- set #7

ktoliver committed:
fix metadata formatting

fix metadata formatting

v-shils committed:
Merge pull request #36391 from mmacy/aci-auth-with-acr

[ACI] deploy from ACR w/service principal

ktoliver committed:
Merge pull request #37391 from bandersmsft/master

cost mgt - updated connect aws account

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-0321

Commit:
Merge pull request #37397 from CarlRabeler/master

latency

ktoliver committed:
Merge pull request #37398 from daveba/tutorial-linux-vm-access-cosmos-db

[Azure AD MSI] Access Azure Cosmos DB on Linux VM

tamram committed:
adding metadata to include files

ktoliver committed:
Merge pull request #37267 from MarkMcGeeAtAquent/table-remove-includes

Cosmos DB: add [includes] around services, remove [includes] about Table concepts

Commit:
Merge pull request #37394 from jimdial/patch-139

Added in-page links for portal.

Commit:
Merge pull request #37393 from jimdial/patch-138

Added in-page links for portal.

grlima committed:
Update application-insights-correlation.md

Commit:
Added PowerShell link.

tfitzmac committed:
fixed intro

ktoliver committed:
Merge pull request #37297 from mattbriggs/mdb_20180406_pki_cert

update self-northwest host

PRMerger15 committed:
Merge pull request #37361 from grlima/master

AppInsights Correlation: add section on Java SDK

PRMerger15 committed:
Merge pull request #37401 from jimdial/patch-140

Added PowerShell link.

v-shils committed:
Merge pull request #35939 from tamram/tamram-0321

adding note about page blob restriction - support issue

PRMerger16 committed:
Merge pull request #37374 from barbkess/casey

fixes from casey

tracsman committed:
Merge master fix conflict

Commit:
removing extra column

PRMerger19 committed:
Merge pull request #37384 from mimig1/04092018meta15

Cosmos DB: Correcting metadata in Mimi's files

Commit:
separating into 2 columns

PRMerger20 committed:
Merge pull request #37372 from mimig1/04092018meta

Cosmos DB: Moving Mimi's files to Sneha & Kelly (11)

douglaslMS committed:
Made change to Roles section rec'd from PM.

PRMerger20 committed:
Merge pull request #37383 from mimig1/04092018meta14

Cosmos DB: Moving Mimi's files to Sneha & Kelly (14)

douglaslMS committed:
Updated ms.date.

Commit:
Merge pull request #37403 from tracsman/jonorTemp

Update content for new features and fix sentance for clarity

Commit:
assigning to writer who's currently handling data lake stuff

Commit:
Updated based on customer feedback

PRMerger12 committed:
Merge pull request #37404 from douglaslMS/adf-fix-roles

Made change to Roles section rec'd from PM.

PRMerger12 committed:
Merge pull request #37249 from cabailey/cabailey-azuredocs

Add details about takeover restrictions for RMS for individuals if in…

tfitzmac committed:
fixed title for CLI

tfitzmac committed:
fixed title for PowerShell

PRMerger13 committed:
Merge pull request #37407 from Blackmist/freshness

assigning to writer who's currently handling data lake stuff

JasonWHowell committed:
Update stream-analytics-machine-learning-anomaly-detection.md

JasonWHowell committed:
Update stream-analytics-machine-learning-anomaly-detection.md

PRMerger14 committed:
Merge pull request #37408 from jeffgilb/49.1

Updated based on customer feedback

JasonWHowell committed:
Update stream-analytics-machine-learning-anomaly-detection.md

Commit:
fixed marketplace item

ArLucaID committed:
Update Identity Assign Command

PRMerger18 committed:
Merge pull request #37360 from areddish/update-install

Update install instructions

Commit:
date touched

PRMerger19 committed:
Merge pull request #37409 from JasonWHowell/master

Fixes MicrosoftDocs/azure-docs#6591

ArLucaID committed:
Update CLI commands

PRMerger20 committed:
Merge pull request #37410 from jeffgilb/49.2

fixed marketplace item

Commit:
Merge pull request #37414 from arluca/patch-8

Update Identity Assign Command

mimig1 committed:
Cosmos DB: Moving arramac's file to Sneha

Commit:
Merge pull request #37416 from arluca/patch-9

Update CLI commands

Commit:
Merge pull request #37417 from dan-wesley/week2-test-branch

date touched

mimig1 committed:
Cosmos DB: Moving arramac's files to Sneha

mimig1 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 04092018meta16

Ja-Dunn committed:
Merge pull request #37198 from tfitzmac/0405scripts

added cli and powershell samples

mimig1 committed:
Cosmos DB: Moving arramac's file to Sneha.

Commit:
Merge pull request #37385 from esung22/update

Update

bandersmsft committed:
cost mgt - updated user access tutorial for fresh

jodebrui committed:
region limitations for vCore model

rwike77 committed:
Freshness

CarlRabeler committed:
[sql db] linked servers

jodebrui committed:
Update sql-database-service-tiers.md

rwike77 committed:
Freshness

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

jodebrui committed:
Update sql-database-service-tiers.md

PRMerger16 committed:
Merge pull request #37429 from rwike77/patch-3

Freshness

PRMerger16 committed:
Merge pull request #37427 from rwike77/patch-2

Freshness

mimig1 committed:
corrections

PRMerger16 committed:
Merge pull request #37406 from cynthn/columns

Columns

PRMerger16 committed:
Merge pull request #37428 from CarlRabeler/master

[sql db] linked servers

PRMerger17 committed:
Merge pull request #37423 from mimig1/04092018meta18

Cosmos DB: Moving arramac's file to Sneha.

Ja-Dunn committed:
Merge pull request #37419 from mimig1/04092018meta15

Cosmos DB: Moving arramac's file to Sneha

v-alje committed:
Merge pull request #37431 from MicrosoftDocs/FromPublicRepo

From public repo

rjagiewich committed:
Merge pull request #37425 from bandersmsft/master

cost mgt - updated user access tutorial for fresh

PRMerger20 committed:
Merge pull request #37422 from mimig1/04092018meta16

Cosmos DB: Moving arramac's files to Sneha

PRMerger20 committed:
Merge pull request #37426 from jodebrui/patch-12

region limitations for vCore model

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
